tests/test_border_pad.py

```diff
@@ -51,6 +51,23 @@ def test_pad_shape(self, input_param, input_data, expected_val):
         result = padder(input_data, mode=input_param["mode"])
         self.assertAlmostEqual(result.shape, expected_val.shape)
 
+    def test_pad_kwargs(self):
+        padder = BorderPad(
+            spatial_border=2,
+            mode="constant",
+            constant_values=(
+                (
+                    0,
+                    0,
```

Comment:
```suggestion
                    0
```

Reply:
updated in the latest commit.

Thanks.

Reply:
Thanks for your fix, I misunderstood your comments, I thought you want to use `(0)` to replace `(0, 0)`..